build/Common.nonprod.props

```diff
@@ -42,6 +42,7 @@
     <MicrosoftExtensionsLoggingAbstractionsPkgVer>[6.0.0,)</MicrosoftExtensionsLoggingAbstractionsPkgVer>
     <MicrosoftNETTestSdkPkgVer>[17.4.1]</MicrosoftNETTestSdkPkgVer>
     <MoqPkgVer>[4.18.3,5.0)</MoqPkgVer>
+    <PersistentStoragePkgVer>1.0.0-beta.1</PersistentStoragePkgVer>
```

Comment:
we'll need to stage this after 1.4.0 ships, as we need bake time + need stable dependencies.
Alternatively we need to use a separate branch. (we already have logs and metrics branch, so might be good to piggy back on that).

Reply:
Branch to merge is now [open-telemetry:main-metrics](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main-metrics)